docs/deploying-site/README.md

```diff
@@ -80,11 +80,27 @@ But if you want your documentation to be accessiable to **anyone** with a github
 ```yml
 steps:
   - name: Build documentation site
-    uses: bcc-code/bcc-documentation-base@v4
+    uses: bcc-code/bcc-documentation-base@v5
     with:
       ...
       public: true
 ```  
 ::: warning <del></del>
 Be sure you are using **v3** or later of the action.
+:::
+
+#### Change authentication method for the documentation
+By default, the documentation is only accessible by logging in using a github account.  
+But if you want your documentation to be accessible using a different provider, than it can be configured using the `authentication` option in the action:
+```yml
+steps:
+  - name: Build documentation site
+    uses: bcc-code/bcc-documentation-base@v5
+    with:
+      ...
+      authentication: 'azuread'
+```  
+- Currently the only available providers are: **Github**, **AzureAD** and **BCC Portal**.  
+::: warning <del></del>
```

Comment:
It's a nitpick, but I see you're using this to delete the title. There doesn't seem to be a way to otherwise delete the title, but I would suggest to use another element that `del`, as it has a [specific meaning](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del). This might impact people that use a screen reader for example (it's a stretch and will probably be ok, but it's good to keep in mind). 

I would suggest to use an element that has no semantic meaning, such as `span`:

```suggestion
::: warning <span></span>
```